go/vt/discovery/topology_watcher.go

```diff
@@ -70,7 +67,7 @@ type TopologyWatcher struct {
 	cell                string
 	refreshInterval     time.Duration
 	refreshKnownTablets bool
-	concurrency         int
+	concurrency         int64
```

Comment:
This field is now unused. It is only written to once and never read.

Reply:
@GuptaManan100 thanks, I've cleaned that up in https://github.com/vitessio/vitess/pull/17276/commits/2f31a570234a17928dc70ff01ecb257f1e6e3f5c 👍